tests/wpt/meta/css/css-fonts/font-variation-settings-descriptor-03.html.ini

```diff
@@ -0,0 +1,2 @@
+[font-variation-settings-descriptor-03.html]
+  expected: FAIL
```

Comment:
These two tests fail due to the way we treat `font-optical-size:auto`. Apparently other browsers don't add `opsz` if its already present in the `font-variation-settings` property.